src/components/GridItem/GridItem.stories.tsx

```diff
@@ -0,0 +1,55 @@
+// importing boilerplate stuff for Storybook
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+// importing your new component
+import GridItem from './GridItem';
+import Grid from '../Grid/Grid';
+
+// title: group / name in Storybook
+// component: your new component
+// argTypes: onClick is a 'Storybook Event' to simulate clicks
+export default {
+    title: '3.Layout/GridItem',
+    component: GridItem,
+    argTypes: {},
+} as ComponentMeta<typeof GridItem>;
+
+// another boilerplate to set up your Storybook template with your new component
+const Template: ComponentStory<typeof GridItem> = (args) => (
+    <Grid
+        gridColumns={3}
+        gridRows={3}
+        responsive={false}
+        responsiveProps={{
+            minWidth: 50,
+            minHeight: 50,
+            maxWidth: '1fr',
+            maxHeight: '1fr',
+            responsiveType: 'auto-fit',
+            enableAuto: false,
+        }}
+        disableTestChildren={true}
+    >
+        <GridItem {...args} />
+    </Grid>
+);
+
+// ////////////////////////////////////////////////////////////////////////////
+// NOTE please only include the min props needed for each render
+// this means the next dev using 'get code' in Storybook will not get confused
+// ////////////////////////////////////////////////////////////////////////////
+
+// Story 1 Default
+export const Sample = Template.bind({});
+Sample.args = {
+    gridArea: [2, 2, 3, 3],
+    children: [
```

Comment:
Does this GridArea prop work ? 
I tried to use it together with Grid component but it was not working like i wanted it to 🤔  (Maybe i am doing something wrong 😓 )
Could you add a story using this in Grid.stories.tsx file

Reply:
Yes. It works. I added a story under GridItem > Sample. In this, I used fixed width to view the box.
![gridItem test](https://user-images.githubusercontent.com/15834299/178146075-e138c47c-baed-4158-b15d-45ce7baf012b.gif)

Under Grid Sample Layout stories, I used Grid Item to build the samples.

Reply:
Ohh cool, i guess i was doing something wrong 😅 
Thanks for clarifying 🙌🏻